src/platform/forms-system/src/js/widgets/PhoneNumberWidget.jsx

```diff
@@ -30,7 +30,7 @@ export default class PhoneNumberWidget extends React.Component {
     return (
       <TextWidget
         {...this.props}
-        inputType={'tel'}
+        type={'tel'}
```

Comment:
Looks like `TextWidget` takes an `options.inputType` prop, but not an `inputType` prop. Good catch. :+1: